app/views/podcasts/show.rss.builder

```diff
@@ -87,6 +87,7 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
 
         xml.itunes :title, ep.clean_title unless ep.clean_title.blank?
         xml.itunes :subtitle, ep.subtitle unless ep.subtitle.blank?
+        # NOTE: you'll only get a tag if this was explicitly set (pun intended)
         xml.itunes :explicit, ep.explicit unless ep.explicit.blank?
```

Comment:
This does _not_ fallback to the podcast tag right now.  Which is a bit contrary to @kookster's comments - that he'd prefer this tag always be present.

I think right now it _looks_ like the field is always present in our feeds because Publish explicitly sets this on every Episode.  (Setting it to the same value as the podcast - but not leaving it nil).